File: v2x_net45_FSharp/BasicTests.fs
namespace FSharp

open Xunit

type BasicTests() =
    [<Fact>]
    member x.Passing() =
        Assert.True(true)

    [<Fact>]
    member x.Failing() =
        Assert.True(false)

    [<Theory>]
    [<InlineData("hello")>]
    [<InlineData("")>]
    member x.Theory(value:string) =
        Assert.NotEmpty(value)

File: console/v2x_net472_Sdk/BasicTests.cs
using Xunit;

public class BasicTests
{
    [Fact]
    public void Passing()
    {
        Assert.True(true);
    }

    [Fact]
    public void Failing()
    {
        Assert.True(false);
    }

    [Theory]
    [InlineData(2112)]
    [InlineData("hello from v2x_net472_Sdk")]
    [InlineData(null)]
    public void Theory(object value)
    {
        Assert.NotNull(value);
    }
}
